src/components/input/input-animations.spec.js

```diff
@@ -1,70 +1,73 @@
 describe('md-input-container animations', function() {
-  var $rootScope, $compile, $animateCss, $material, $$mdInput,
-    el, pageScope, invalidAnimation, messagesAnimation, messageAnimation,
-    cssTransitionsDisabled = false, lastAnimateCall;
+  var $rootScope, $compile, $material, $$mdInput, $window, $animate, $rootElement, $document, $timeout,
+    el, root, body, pageScope, computedStyle;
 
   // Load our modules
   beforeEach(module('ngAnimate', 'ngMessages', 'material.components.input', 'material.components.checkbox'));
 
   // Run pre-test setup
-  beforeEach(decorateAnimateCss);
   beforeEach(injectGlobals);
   beforeEach(setupVariables);
 
   // Run after-test teardown
   afterEach(teardown);
 
-  it('set the proper styles when showing messages on an input', function() {
+  it('set the proper styles when showing messages on an input', performInputAnimationTests);
+  it('set the proper styles when showing messages on an input with animations disabled', function() {
+    $animate.enabled(false);
+    performInputAnimationTests();
+    $animate.enabled(true);
+  });
+
+  function performInputAnimationTests() {
     compile(
       '<form name="testForm">' +
       '  <md-input-container>' +
       '    <input name="foo" ng-model="foo" required ng-pattern="/^1234$/" />' +
       '    <div class="errors" ng-messages="testForm.foo.$error">' +
-      '      <div ng-message="required">required</div>' +
-      '      <div ng-message="pattern">pattern</div>' +
+      '      <div ng-message="required" style="transition: 0s none">required</div>' +
+      '      <div ng-message="pattern" style="transition: 0s none">pattern</div>' +
```

Comment:
This is the same method of testing as with the original code which had this block:

```
// Make sure any transitions happen immediately; NOTE: this is REQUIRED for the above
// tests to pass without using window.setTimeout to wait for the animations
if (cssTransitionsDisabled) {
  element.css('transition', '0s none');
}
```

I just moved it inline since I dropped animation call tracing in favor or computed style checking